key,default_value
additionalEventData_SignatureVersion,"SigV4"
additionalEventData_CipherSuite,"ECDHE-RSA-AES128-GCM-SHA256"
additionalEventData_AuthenticationMethod,"AuthHeader"
additionalEventData_x-amz-id-2,"i09xXQ2MKcrLDhQzAfnGC/gLINoTjDo3Y3hG9TIUiAY7xRnJ6F5kileIlo3KWscZRstLfHChN5Q="
userAgent,"[Boto3/1.14.51 Python/3.8.5 Linux/4.19.76-linuxkit Botocore/1.17.51]"
eventID,"fee449117-f1bd-49c5-a24a-f9b36127b272"
errorMessage,"The specified bucket does not exist"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDADO2GQD0K8TEF7KW1V"
userIdentity_arn,"arn:aws:iam::811596193553:user/Level6"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA3Z2XBVUDFQ9TU4MD"
userIdentity_userName,"Level6"
eventType,"AwsApiCall"
errorCode,"NoSuchBucket"
sourceIPAddress,"9.240.250.1"
eventName,"GetBucketAnalyticsConfiguration"
eventSource,"s3.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters_analytics,""
requestParameters_bucketName,"dummy_data"
requestParameters_Host,"s3.amazonaws.com"
requestParameters_id,"dummy_data"
awsRegion,"us-east-1"
readOnly,"True"
requestID,"0P5Z2XBQAWCN2PDP"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2020-09-21T04:28:01Z"
